infor-design/enterprise
# 3120 / 3118 - Fix a few weekview bugs [v4.23.x]

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Fixes two straightforward QA bugs and one more i found that the month text wasnt updating after last PR. Please see test steps.

**Related github/jira issue (required)**:
Fixes #3210
Fixes #3118 

**Steps necessary to review your pull request (required)**:
**Test One**
- go to http://localhost:4000/components/week-view/example-two-day.html
- click today (2 days should still be shown)
- go back and forth (2 days should still be shown)
- go forward or back enough times so the month should change (month should update)

**Test Two**
- go to http://localhost:4000/components/week-view/example-index.html?theme=soho&variant=dark&colors=2578a9
- in the month picker - select the month of Oct 20th , 2019 by clicking any day that week
- the icon in the calendar should be visible
- switch all themes
- icon should look fine.
